src/hotspot/share/services/memTracker.hpp

```diff
@@ -137,14 +137,9 @@ class MemTracker : AllStatic {
     return _tracking_level;
   }
 
-  // Shutdown native memory tracking.
-  // This transitions the tracking level:
-  //  summary -> minimal
-  //  detail  -> minimal
-  static void shutdown();
-
-  // Transition the tracking level to specified level
-  static bool transition_to(NMT_TrackingLevel level);
+  static inline bool enabled() {
+    return _tracking_level >= NMT_summary;
```

Comment:
```suggestion
    return _tracking_level > NMT_off;
```

Same reason as above.